fhttp/http_utils.go

```diff
@@ -627,3 +627,11 @@ func (r *SyncReader) Read(p []byte) (n int, err error) {
 
 	return r.reader.Read(p)
 }
+
+func getIPUsageCount(ipCountMap map[string]int) (count int) {
```

Comment:
this should only be needed in the test so should be in the test file?

Reply:
moved to httprunner_test.go